src/parse_file.cpp

```diff
@@ -0,0 +1,19 @@
+#include "parse_file.hpp"
+
+namespace docgen {
+
+using namespace core;
```

Comment:
move this inside `parse_file` definition. It's better practice

Reply:
good to know :)